CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # CHANGELOG
 
+## v0.5.0
+
+### What's New
+
+**Manage EdgeX Foundry system in OpenYurt in a cloud-native, non-intrusive way**
+
+- yurt-edgex-manager
+  Yurt-edgex-manager enable OpenYurt to be able to manage the EdgeX lifecycle. Each EdgeX CR (Custom Resource) stands for an EdgeX instance.
+Users can deploy/update/delete EdgeX in OpenYurt cluster by operate the EdgeX CR directly. ([yurt-edgex-manager](https://github.com/openyurtio/yurt-edgex-manager), [@yixingjia](https://github.com/yixingjia), [@lwmqwer](https://github.com/lwmqwer))
+
+- yurt-device-controller
+  Yurt-device-controller introduces an approach leverages existing edge computing platforms, like EdgeX Foundry, and uses Kubernetes custom resources to manage edge devices.
+These CRDs and corresponding controllers allow users to manage edge devices in a declarative way, which provides users with a Kubernetes-native experience and reduces the complexity of managing,
+operating and maintaining edge platform devices. ([yurt-device-controller](https://github.com/openyurtio/yurt-device-controller), [@qclc](https://github.com/qclc), [@Peeknut](https://github.com/Peeknut), [@rambohe-ch](https://github.com/rambohe-ch), [@yixingjia](https://github.com/yixingjia))
```

Comment:
I think we should add Charles Zhang as contributor as well.

Reply:
sorry, @charleszheng44 is missed.

Reply:
added.